projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -18,13 +18,11 @@ export class CartEffects {
     ofType(fromActions.LOAD_CART, LANGUAGE_CHANGE, CURRENCY_CHANGE),
     map((action: any) => action.payload),
     mergeMap(payload => {
```

Comment:
we should change the name of this `payload` param to something else, due to better readability

Reply:
updated